crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -4508,3 +4573,35 @@ fn inner_size(
 ) -> TaoPhysicalSize<u32> {
   window.inner_size()
 }
+
+fn get_work_area_size(target_monitor: &MonitorHandle) -> TaoPhysicalSize<u32> {
```

Comment:
we should create a `MonitorExt` and implement it on `MonitorHandle`, all in a separate file, see `tauri-runtime-wry/src/window/mod.rs` for an example with `WindowExt`